docs/source/features/moe.rst

```diff
@@ -4,7 +4,7 @@ Mixture of Experts
 Overview
 --------
 
-NeMo supports Mixture of Experts (MoE) in the transformer layer for NLP models.
+NeMo supports Mixture of Experts (MoE) in the feedforward block of the transformer layer.
 
 MoE is a machine learning technique where multiple specialized models (experts,
```

Comment:
Line 8. Move Overview heading here like this:

Overview
-----------

Line 14. Add a heading.

Use Moe
----------


Line 28. Add a heading.

Configure MoE-specific Loss Function
------------------------------------------